docs/platforms/javascript/common/tracing/distributed-tracing/dealing-with-cors-issues/index.mdx

```diff
@@ -16,8 +16,53 @@ notSupported:
   - javascript.nestjs
 ---
 
-If your frontend and backend are hosted on different domains (for example, your frontend is on `https://example.com` and your backend is on `https://api.example.com`), and the frontend does XHR/fetch requests to your backend, you'll need to configure your backend [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Headers) headers to ensure requests aren't blocked.
+If your frontend and backend are hosted on different domains (for example, your frontend is on `https://example.com` and your backend is on `https://api.example.com`), you need to configure your backend [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Headers) headers to prevent requests from being blocked by the browser.
 
-Configure your backend CORS to allow the `sentry-trace` and `baggage` headers.
+## Understanding Trace Propagation
 
-Your server's response header configuration might look like: `"Access-Control-Allow-Headers: sentry-trace, baggage"`. Your configuration will be specific to your setup.
+Sentry doesn't attach the `sentry-trace` and `baggage` headers to every outgoing request. Instead, it only attaches these headers to requests whose URLs match the patterns specified in the `tracePropagationTargets` configuration option.
+
+**By default**, `tracePropagationTargets` is set to `['localhost', /^\//]`, which means trace headers are only attached to:
```

Comment:
this is the default for browser. For node, all requests will be propagated by default!

Reply:
hmm, or this page is only visible on browser anyhow I suppose, so all good :)